typescript-playwright-sample/README.md

```diff
@@ -31,23 +31,44 @@ Some good places to start reading are:
 
 ## See it in action in Azure Pipelines
 
-[![Build Status](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_apis/build/status/42?branchName=main)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/latest?definitionId=42&branchName=main)
-
 <!--
   Note to maintainers: The below example images/links come from a specific build instead of the most recent build so we can link to specific tabs.
   If you update the links such that they point to a different build, make sure to mark that build as Retained so the links don't expire in a month.
 -->
-The accessibility tests run as part of the `yarn test` build step:
 
-[![Screenshot of "yarn test" build logs in sample build](./assets/screenshot-logs-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=22505)
+These two azure pipelines demonstrate what it looks like to incorporate passing or failing accessibility tests into your CI.
+
+### Passing examples
+
+[![Build Status](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_apis/build/status/42?branchName=main)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/latest?definitionId=42&branchName=main)
+
+The passing accessibility tests included in passing-examples.spec.ts run as part of the `yarn test` build step:
+
+[![Screenshot of "yarn test" build logs in passing sample build](./assets/screenshot-passing-logs-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=39169)
+
+The test pass results display in the Tests tab of the build logs:
+
+[![Screenshot of Tests tab in passing sample build](./assets/screenshot-passing-tests-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=39169&view=ms.vss-test-web.build-test-results-tab)
+
+Detailed accessibility scan information also appears in the Scans tab, courtesy of the [SARIF SAST Scans Tab extension](https://marketplace.visualstudio.com/items?itemName=sariftools.scans):
+
+[![Screenshot of Scans tab in passing sample build](./assets/screenshot-passing-scans-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=39169&view=sariftools.sarif-viewer-build-tab.sariftools.sarif-viewer-build-tab)
```

Comment:
It looks like the required `view` param changed when the sarif viewer extension got renamed, and we missed updating these links. I think this link should instead be:

```suggestion
[![Screenshot of Scans tab in passing sample build](./assets/screenshot-passing-scans-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=39169&view=sariftools.scans.build-tab)
```

(same for the failing sample build link on L71, and also in the typescript-selenium sample readme)